bootstrapvz/plugins/file_copy/__init__.py

```diff
@@ -3,13 +3,19 @@
 
 def validate_manifest(data, validator, error):
 	import os.path
+	from bootstrapvz.common.exceptions import TaskError
+
 	schema_path = os.path.normpath(os.path.join(os.path.dirname(__file__), 'manifest-schema.yml'))
 	validator(data, schema_path)
 
+	for file_entry in data['plugins']['file_copy']['files']:
+		srcfile = file_entry['src']
+		if not os.path.isfile(srcfile):
+			msg = 'The source file %s does not exist.' % srcfile
+			raise TaskError(msg)
```

Comment:
Use the `error` function provided as an argument instead. [Like so](https://github.com/andsens/bootstrap-vz/blob/e961438904fdbe7c88e4c8d20cc211c5026e5a5c/bootstrapvz/providers/ec2/__init__.py#L32)